tests/Serializer/XmlSerializationTest.php

```diff
@@ -385,15 +387,6 @@ public function testDeserializingNull()
         $this->markTestSkipped('Not supported in XML.');
     }
 
-    public function testDeserializeWithObjectWithToStringMethod()
```

Comment:
Objects with `__toString()` are not `string`, therefore removed as incompatible with strict types.

Reply:
ok, this "feature" can be removed

Reply:
It has simple user-land workaround: `(string) $stringishObject`, but imho that doesn't make much sense anyway. :grinning: